app/controllers/onboarding_controller.rb

```diff
@@ -17,28 +17,42 @@ def create
     return redirect_to_success if User.email_taken?(user_params[:email])
 
     @user = User.new(user_params)
-    return redirect_to_success if @user.save
+    if @user.save
+      JsonWebToken.create(invalidated_jti: params[:jwt])
+      return redirect_to_success
+    end
 
     render :index
   end
 
   def success; end
 
+  def create_invite_url
+    payload = SecureRandom.base64(16)
+    jwt = JsonWebToken.encode(payload)
+    render json: { url: onboarding_url(jwt: jwt) }
+  end
+
   private
 
   def redirect_to_success
-    redirect_to onboarding_success_path(token: token_param)
+    redirect_to onboarding_success_path(jwt: jwt_param)
   end
 
   def verify_token
-    raise ActionController::BadRequest unless token_param == ENV['ONBOARDING_TOKEN']
+    invalidated_jti = JsonWebToken.where(invalidated_jti: jwt_param)
```

Comment:
@mattwr18 Why is the column called `invalidated_jti`? From what I can see, it stores the complete token.

Reply:
yeah, that's true, I think it doesn't make a big difference either way, but it's nice to keep our semantics more accurate.